apps/website/src/content/docs/table.mdx

```diff
@@ -6,16 +6,466 @@ thumbnail: Table
+            >
+              <SvgMore />
+            </IconButton>
+          </DropdownMenu>
+        );
+      },
+    },
+  ],
+  [],
+);
+```
+
+## Sorting
+
+To enable sorting, set `isSortable` to `true`. To handle sorting manually (e.g. on the server-side), also pass `manualSortBy={true}` to `<Table>`. Can use in conjunction with `onSort` or `stateReducer` to listen to sorting changes.
+
+Sort arrows (↑ for ascending and ↓ for descending) are displayed in the header cells according to the following:
+
+- If a column is used for sorting, arrows are displayed regardless of hover.
+- If a column is not used for sorting but is sortable, the arrow appears only on hover to indicate the type of sorting that _will_ apply when clicked.
+- If a column is not sortable, no arrow will appear at all.
+
+Column specific props:
+
+- First sort option is ascending. To make descending as the first choice, pass `sortDescFirst: true` to the column object.
+- To disable sorting for a column, pass `disableSortBy: true` to the column object.
+
+```jsx {7, 12, 21}
+const columns = React.useMemo(
+  () => [
+    /* … */
+    {
+      Header: 'Name',
+      accessor: 'name',
+      sortDescFirst: true,
+    },
+    {
+      Header: 'Description',
+      accessor: 'description',
+      disableSortBy: true,
+    },
+  ],
+  [],
+);
+
+<Table
+  /* … */
+  columns={columns}
+  isSortable
+/>;
+```
+
+<LiveExample src='Table.sorting.jsx'>
+  <AllExamples.TableSortingExample client:load />
+</LiveExample>
+
+## Subrow
+
+The `Table` component supports hierarchical data structures allowing end users to display collapsible subrows. Each data entry in the `data` array can have a `subRows` array.
+
+```jsx {6, 11}
+const data = React.useMemo(() => {
+  return [
+    {
+      name: 'Row 1',
+      description: 'Description',
+      subRows: [{ name: 'Subrow 1', description: 'Description 1' }],
+    },
+    {
+      name: 'Row 2',
+      description: 'Description',
+      subRows: [
+        {
+          name: 'Subrow 2',
+          description: 'Description 2',
+          subRows: [{ name: 'Subrow 2.1', description: 'Description 2.1' }],
+        },
+      ],
+    },
+  ];
+}, []);
+```
+
+<LiveExample src='Table.subRows.jsx'>
+  <AllExamples.TableSubRowsExample client:load />
+</LiveExample>
+
+## Expandable content
+
+**Note**: Subrows and Expandable content _cannot_ be used simultaneously.
+
+Apart from collapsible subrows, custom collapsible content can also be rendered right after the row if it is expanded. This is done by passing a function to the `subComponent` prop.
+
+<LiveExample src='Table.expandableContent.jsx'>
+  <AllExamples.TableExpandableContentExample client:load />
+</LiveExample>
+
+## Virtualization
+
+**Note**: This feature is still considered _experimental_ and so may have some performance issues, bugs, future changes, etc. If virtualization does not work, [pagination](#pagination) is the next best alternative for large datasets.
```

Comment:
This note can be moved after the demo.

Reply:
Then place it before the demo but after the main text. It doesn't make sense to start a section with "this feature is still considered experimental" when we haven't explained what the feature is.

Also, when the note is the first paragraph, it creates ambiguity around where the note ends ("Is the second paragraph part of the note or not?"). We could fix this with styling, but semantically it would still remain ambiguous. So as a rule of thumb, notes should be placed at the end.